File: scratch/query_planners.js
import { createClient } from '@supabase/supabase-js';
import dotenv from 'dotenv';
import fs from 'fs';

// Load .env.local
const envConfig = dotenv.parse(fs.readFileSync('.env.local'));
const supabaseUrl = envConfig.VITE_SUPABASE_URL;
const supabaseKey = envConfig.SUPABASE_SERVICE_ROLE_KEY;

if (!supabaseUrl || !supabaseKey) {
  console.error("Missing supabase URL or service role key in .env.local");
  process.exit(1);
}

const supabase = createClient(supabaseUrl, supabaseKey);

async function run() {
  const { data, error } = await supabase.from('planners').select('*');
  if (error) {
    console.error("Error fetching planners:", error);
  } else {
    console.log("Planners count:", data.length);
    data.forEach(p => {
      console.log(`ID: ${p.id} | Code: ${p.planner_code} | Name: ${p.name} | Cert: ${p.certification_message}`);
    });
  }
}

run();

File: scratch/query_planners.py
import os
import dotenv
from supabase import create_client

# Load env variables from .env.local
env = dotenv.dotenv_values(".env.local")
url = env.get("VITE_SUPABASE_URL")
key = env.get("SUPABASE_SERVICE_ROLE_KEY")

if not url or not key:
    print("Env missing URL or key!")
    exit(1)

supabase = create_client(url, key)
response = supabase.table("planners").select("*").execute()
for planner in response.data:
    print(f"ID: {planner.get('id')} | Code: {planner.get('planner_code')} | Name: {planner.get('name')}")
